test/integration/default/controls/audit_spec.rb

```diff
@@ -0,0 +1,43 @@
+# encoding: utf-8
```

Comment:
It would be great to have (somewhere) a dev doc about testing the audit cookbook.  It could be comments here, but I'd look at docs/dev/ first.